src/upgrade/upgrade_scripts/5.3.0/update_system_default_chunk_config.js

```diff
@@ -0,0 +1,28 @@
+/* Copyright (C) 2016 NooBaa */
+"use strict";
```

Comment:
@jeniawhite the file is placed in a 5.3.0 folder. this means that it will only run when upgrading from a version that is earlier than 5.3.0. if the upgrade script is for 5.5 it needs to be in a 5.5.0 folder. 

maybe we should think of adding a `latest` folder as the default location to place new upgrade scripts. we will then need to move the content after every release. e.g after releasing 5.5.0 we will move all scripts in latest to `5.5.0` folder. in any case it's not for this PR